diff --git b/sdk/vault-client/src/generated/models/IAMConfig.ts a/sdk/vault-client/src/generated/models/IAMConfig.ts
index bfd0769..a4aaa38 100644
--- b/sdk/vault-client/src/generated/models/IAMConfig.ts
+++ a/sdk/vault-client/src/generated/models/IAMConfig.ts
@@ -2,14 +2,5 @@
 /* istanbul ignore file */
 /* tslint:disable */
 /* eslint-disable */
-import type { Idp } from './Idp';
-import type { Policy } from './Policy';
-import type { Role } from './Role';
-import type { User } from './User';
-export type IAMConfig = {
-  policies?: Record<string, Policy>;
-  roles?: Record<string, Role>;
-  users?: Record<string, User>;
-  idps?: Record<string, Idp>;
-};
+export type IAMConfig = string;
 
