diff --git b/sdk/vault-client/src/generated/models/IAMConfig.ts a/sdk/vault-client/src/generated/models/IAMConfig.ts
index cc0c580..c905d4b 100644
--- b/sdk/vault-client/src/generated/models/IAMConfig.ts
+++ a/sdk/vault-client/src/generated/models/IAMConfig.ts
@@ -3,15 +3,5 @@
 /* tslint:disable */
 /* eslint-disable */
 
-import type { Idp } from './Idp';
-import type { Policy } from './Policy';
-import type { Role } from './Role';
-import type { User } from './User';
-
-export type IAMConfig = {
-  policies?: Record<string, Policy>;
-  roles?: Record<string, Role>;
-  users?: Record<string, User>;
-  idps?: Record<string, Idp>;
-};
+export type IAMConfig = string;
 
