diff --git a/sdk/vault-client/src/generated/models/InputObject.ts b/sdk/vault-client/src/generated/models/InputObject.ts
index 4878340..fde41b3 100644
--- a/sdk/vault-client/src/generated/models/InputObject.ts
+++ b/sdk/vault-client/src/generated/models/InputObject.ts
@@ -16,8 +16,9 @@ export type InputObject = {
   /**
    * The ID of an object.
    */
-  id?: string;
-  fields?: ObjectFields;
-  encrypted?: EncryptedObjectInput;
+  id: string;
+} | {
+  fields: ObjectFields;
+} | {
+  encrypted: EncryptedObjectInput;
 };
-
