diff --git b/sdk/vault-client/src/generated/models/InputObject.ts a/sdk/vault-client/src/generated/models/InputObject.ts
index 3b043f7..9f7f45b 100644
--- b/sdk/vault-client/src/generated/models/InputObject.ts
+++ a/sdk/vault-client/src/generated/models/InputObject.ts
@@ -19,12 +19,15 @@ export type InputObject = {
   /**
    * The ID of an object.
    */
-  id?: string;
-  fields?: ObjectFields;
-  encrypted?: EncryptedObjectInput;
+  id: string;
+} | {
+  fields: ObjectFields;
+} | {
+  encrypted: EncryptedObjectInput;
+} | {
   /**
    * The index of the object in the request array.
    */
-  request_index?: number;
+  request_index: number;
 };
 
