diff --git a/sdk/vault-client/src/generated/models/ObjectFields.ts b/sdk/vault-client/src/generated/models/ObjectFields.ts
index 9477792..06dd12e 100644
--- a/sdk/vault-client/src/generated/models/ObjectFields.ts
+++ b/sdk/vault-client/src/generated/models/ObjectFields.ts
@@ -6,5 +6,6 @@
  * A list of maps of object properties and their values.
  */
 export type ObjectFields = {
+  [key: string]: any;
 };
 
